code/ATMOSPHERICS/components/unary/vent_scrubber.dm

```diff
@@ -2,6 +2,8 @@
 	icon = 'icons/atmos/vent_scrubber.dmi'
 	icon_state = "map_scrubber"
 
+	req_one_access_txt = "24;10"

```

Comment:
uhm

that's completely different to any and all other machines you hack with a multitool

why should vents and scrubbers be special snowflakes about it